item.rb

```diff
@@ -0,0 +1,29 @@
+require 'date'
+
+class Item
+  attr_accessor :genre, :author
+  attr_reader :publish_date, :label, :archived, :id
+
+  def initialize(publish_date, id: nil, archived: false)
+    @id = id || Random.rand(101..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def label=(label)
+    @label = label
+    label.items << self unless label.items.include?(self)
+  end
+
```

Comment:
The item class should have one Genre, Author, and Label. As you have done for the Label on lines 13 - 16, kindly provide same for the Genre and Author. :wink: